kubernetes/run-unit-tests.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -e
+
+# We must change local_settings.py, because django search for configmap which is added in kubernetes.
+echo "from .testing import *" > "/srv/http/concent_api/concent_api/settings/local_settings.py"
+# We must add host value to testing.py settings to enable communication with database.
+sed -i '/USER/a "HOST":"postgres",' /srv/http/concent_api/concent_api/settings/testing.py
```

Comment:
If you need to modify values in `testing.py` you should just create another settings file that imports `testing` and define `DATABASES` setting in it.